src/Microsoft.PowerShell.Commands.Utility/commands/utility/trace/MshHostTraceListener.cs

```diff
@@ -62,35 +62,8 @@ internal PSHostTraceListener(PSCmdlet cmdlet)
         [SecurityPermission(SecurityAction.LinkDemand)]
         protected override void Dispose(bool disposing)
         {
-#if CORECLR
             base.Dispose(disposing);
-#else
-            try
-            {
-                if (disposing)
-                {
-                    this.Close();
-                }
-            }
-            finally
-            {
-                base.Dispose(disposing);
-            }
-#endif
-        }
-
-#if !CORECLR
-        /// <summary>
-        /// Closes the dialog and then calls the base class Close
-        /// </summary>
-        [SecurityPermission(SecurityAction.LinkDemand)]
-        public override void Close()
-        {
-            // Call the base class close
-
-            base.Close();
```

Comment:
I think the current change is good. The `Dispose(bool disposing)` implementation in this class is better to be kept so that if a sub class of `PSHostTraceListener` can depend on `Dispose(true)` to call `close()`. It's not a big deal, but this `Dispose` implementation indeed does a bit more than the base type `Dispose` implementation.